src/Bicep.Core.UnitTests/Diagnostics/LinterRuleTests.cs

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.Core.Analyzers.Linter;
+using Bicep.Core.Analyzers.Linter.Rules;
+using Bicep.Core.UnitTests.Utils;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Linq;
+
+namespace Bicep.Core.UnitTests.Diagnostics
+{
+    [TestClass]
+    public class LinterRuleTests
```

Comment:
IMHO we should have a single test file per rule

Reply:
@StephenWeatherford You're working on this aspect currently.